G-Research.OpenTelemetry.Processor.Partial/G-Research.OpenTelemetry.Processor.Partial.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
-    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0;netstandard2.0</TargetFrameworks>
```

Comment:
this shouldn't be necessary for the lib - only for the tests

Reply:
build command was failing, but once i removed that command and added net9.0 tests succeeded.